crates/arroyo-connectors/src/kafka/sink/mod.rs

```diff
@@ -171,8 +171,10 @@ impl ArrowOperator for KafkaSinkFunc {
     async fn process_batch(&mut self, batch: RecordBatch, ctx: &mut ArrowContext) {
         let values = self.serializer.serialize(&batch);
 
-        if !ctx.in_schemas[0].key_indices.is_empty() {
-            let k = batch.project(&ctx.in_schemas[0].key_indices).unwrap();
+        if ctx.in_schemas[0].key_indices.is_some() {
```

Comment:
Could this be an if let instead of if/unwrap?

Reply:
yep